### DISCUSSION TITLE
Is there any way to change the text color in button when hovring ?

### DISCUSSION THREAD

**Discussion by sunnymoon-sultan** [TomSchimansky/CustomTkinter#527] [Q&A]
How can I change the text color of the button when hovering on it?

**Comment by Akascape** [comment]
@sunnymoon-sultan It's possible to change the hover color of text of any CTkButton.
Though the direct parameter is not implemented, but you can use this:

```python
button = ctk.CTkButton(master=frame, width=140, height=50, text="TEXT", text_font=("Roboto Medium",20),  text_color="white")
button.grid()
       
# Add these two bindings for button hover
button.bind("<Enter>", lambda event: button.configure(text_color="red")) 
button.bind("<Leave>", lambda event: button.configure(text_color="white"))       
```
<img src="https://user-images.githubusercontent.com/89206401/196462461-e78d142a-4b4f-42f8-adf1-b1443ebd37c3.gif" width=40%>
